packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -467,3 +467,38 @@ export class Store {
     return store.state[facetType];
   }
 }
+
+export function AriaLiveRegion(regionName, elem, assertive = false) {
+
+  function dispatchMessage(message) {
+    // Creates the event to dispatch an event to an aria-live region
+    const registerRegionEvent = new CustomEvent('arialivemessage', {
+      bubbles: true,
+      detail: {
+        regionName,
+        assertive,
+        message
+      }
+    });
+    elem.dispatchEvent(registerRegionEvent);
+  }
+
+  function registerRegion() {
+    // Creates the event to register an aria-live region
```

Comment:
Maybe the comments should be right before the declaration of the function

Reply:
I don't know why I even have comments 😂